custom/icds_reports/utils/aggregation_helpers/awc_infrastructure.py

```diff
@@ -46,6 +46,7 @@ def data_from_ucr_query(self):
         return """
             SELECT
                 DISTINCT awc_id AS awc_id,
+                supervisor_id AS supervisor_id,
```

Comment:
Should this be joined with locations table to lookup supervisor_id from awc_id, or is this good?